numpy/tests/test_numpy_version.py

```diff
@@ -32,3 +32,13 @@ def test_valid_numpy_version():
         res = re.match(version_pattern + dev_suffix + '$', np.__version__)
 
     assert_(res is not None, np.__version__)
+
+
+def test_short_version():
+    # Check numpy.short_version actually exists
+    if np.version.release:
+        assert_(np.__version__ == np.version.short_version,
+        "short_version in release")
```

Comment:
Should be indented to match start of `np.__version__`, same below. The error message also seems incorrect, it is only shown when  `assert_` fails.

Reply:
In fact, it looks like you don't need the `if` as both cases are the same.

Reply:
You are right about indentation ...